Compilation error due to data-default-class

Hi,

I'm now getting the following building `req` `3.13.4`:

```
Network/HTTP/Req.hs:610:11: error: [GHC-39999]
    • No instance for ‘Data.Default.Class.Default NC.TLSSettings’
        arising from a use of ‘def’
    • In the second argument of ‘L.mkManagerSettingsContext’, namely
        ‘def’
      In the expression:
        L.mkManagerSettingsContext (Just context) def Nothing
      In an equation for ‘settings’:
          settings = L.mkManagerSettingsContext (Just context) def Nothing
    |
610 |           def
    |           ^^^

Error: [Cabal-7125]
```

I presume this is because `req`'s dependency for `data-default-class` is not constrained in hackage - could it be fixed for this version, or instead pinned to the previous version for now?

Thanks

--- a/req.cabal
+++ b/req.cabal
@@ -40,7 +40,7 @@ library
         bytestring >=0.10.8 && <0.13,
         case-insensitive >=0.2 && <1.3,
         containers >=0.5 && <0.8,
-        crypton-connection >=0.3 && <0.5,
+        crypton-connection >=0.4.2 && <0.5,
         data-default-class >=0.2.0.0,
         exceptions >=0.6 && <0.11,
         http-api-data >=0.2 && <0.7,

--- a/req.cabal
+++ b/req.cabal
@@ -41,7 +41,7 @@ library
         case-insensitive >=0.2 && <1.3,
         containers >=0.5 && <0.8,
         crypton-connection >=0.3 && <0.5,
-        data-default-class,
+        data-default-class >=0.2.0.0,
         exceptions >=0.6 && <0.11,
         http-api-data >=0.2 && <0.7,
         http-client >=0.7.13.1 && <0.8,

